SK-1788-TRANSLATION-WEB-Emails-MXplan-Outlook-mobile-Android (#7279)
ovh/docs: pages/web_cloud/email_and_collaborative_solutions/mx_plan/email_outlook_app_android_ios/guide.pt-pt.md

---
title: "MX Plan - Configurar a sua conta de e-mail no Outlook para Android"
excerpt: "Descubra como configurar o seu endereço de e-mail MX Plan na aplicação móvel Outlook para Android"
updated: 2024-11-26
---

<style>
.w-400 {
  max-width:400px !important;
}
.h-600 {
  max-height:600px !important;
}
</style>

## Objetivo

As contas MX Plan podem ser configuradas em diferentes softwares de e-mail compatíveis. Isto permite-lhe utilizar o seu endereço de e-mail a partir do dispositivo que preferir. A aplicação Outlook da Microsoft para Android está disponível gratuitamente a partir da Google Play Store.

**Saiba como configurar o endereço de e-mail MX Plan na aplicação móvel Outlook para Android**

> [!warning]
>
> A OVHcloud oferece-lhe serviços cuja configuração, gestão e responsabilidade é da sua responsabilidade. Assim, deverá assegurar o seu bom funcionamento.
>

> > Este guia fornece as instruções necessárias para realizar as operações mais habituais. No entanto, se encontrar dificuldades, recomendamos que recorra a um [parceiro especializado](https://marketplace.ovhcloud.com/c/support-collaboration) e/ou que contacte o editor do serviço. Não poderemos proporcionar-lhe assistência técnica. Para mais informações, consulte a secção " Ir mais longe " deste guia.

Reviewer: ```suggestion
> Este guia fornece as instruções necessárias para realizar as operações mais habituais. No entanto, se encontrar dificuldades, recomendamos que recorra a um [parceiro especializado](https://marketplace.ovhcloud.com/c/support-collaboration) e/ou que contacte o editor do serviço. Não poderemos proporcionar-lhe assistência técnica. Para mais informações, consulte a secção "Quer saber mais?" deste guia.
```

> > Este guia fornece as instruções necessárias para realizar as operações mais habituais. No entanto, se encontrar dificuldades, recomendamos que recorra a um [parceiro especializado](https://marketplace.ovhcloud.com/c/support-collaboration) e/ou que contacte o editor do serviço. Não poderemos proporcionar-lhe assistência técnica. Para mais informações, consulte a secção " Ir mais longe " deste guia.

Reviewer: ```suggestion
> Este guia fornece as instruções necessárias para realizar as operações mais habituais. No entanto, se encontrar dificuldades, recomendamos que recorra a um [parceiro especializado](https://marketplace.ovhcloud.com/c/support-collaboration) e/ou que contacte o editor do serviço. Não poderemos proporcionar-lhe assistência técnica. Para mais informações, consulte a secção "Quer saber mais?" deste guia.
```

![outlook android](images/outlook-app-android-add01.png){.thumbnail .w-400 .h-600}

- **Se uma conta já tiver sido parametrizada**:
    - Pressione o envelope " &#9993; " na parte superior esquerda do seu ecrã.
    - De seguida, prima o botão `+`{.action} na barra vertical à esquerda.
    - Prima `Adicionar uma conta`{.action}.

![outlook android](images/outlook-app-android-add02.png){.thumbnail .w-400 .h-600}

Siga as etapas de instalação, clicando nos separadores abaixo:

> [!tabs]
> **Etapa 1**
>>
>> Introduza o seu endereço de e-mail e prima `Continuar`{.action}.
>>
>> ![outlook android](images/outlook-app-android-add-step01.png){.thumbnail .w-400 .h-600}
>>
> **Etapa 2**
>>
>> Selecione o protocolo de receção, **IMAP**(recomendado) ou **POP3**.
>>
>> ![outlook android](images/outlook-app-android-add-step02.png){.thumbnail .w-400 .h-600}
>>
>> > [!warning]
>> >
>> > Se a janela de escolha do protocolo não aparecer, prima o botão `?` no canto superior direito do ecrã e escolha `Mudar de fornecedor de conta`{.action}. Selecione `IMAP`(recomendado) ou `POP3`.<br>
>> > ![outlook android](images/outlook-app-android-add-step021.png){.thumbnail .w-400 .h-600}
>>

> > **Passo 3 - IMAP**

Reviewer: ```suggestion
> **Etapa 3 - IMAP**
```

>>
>> Na seguinte janela, selecione `Configurações avançadas`{.action} e introduza as seguintes informações:
>>
>> - **Endereço de correio eletrónico**
>> - **Nome completo** : introduza o seu endereço de e-mail completo

> >> - **Description**

Reviewer: ```suggestion
>> - **Descrição**
```

>> - **Servidor de receção de correio SMTP**:<br>- **Nome do host SMTP**: para a **EUROPE**, introduza `smtp.mail.ovh.net` ou `ssl0.ovh.net` . Para a **AMÉRICA/ÁSIA**, introduza `smtp.mail.ovh.ca`<br>- **Port**: 465<br>- **Tipo de segurança**: SSL/TLS<br>- **Nome de utilizador SMTP**: o seu endereço de correio eletrónico completo<br>- **Palavra-passe SMTP** : o do seu endereço de correio eletrónico
>>
>> Para finalizar a configuração, clique no botão " &#10003; "
>>
>> ![outlook android](images/outlook-app-android-add-step03-imap-eu.png){.thumbnail .w-400 .h-600}
>>
> **Etapa 3 - POP3**
>>
>> Na seguinte janela, selecione `Configurações avançadas`{.action} e introduza as seguintes informações:
>>
>> - **Endereço de correio eletrónico**
>> - **Nome completo** : Introduza o seu endereço de e-mail completo

> >> - **Description**

Reviewer: ```suggestion
>> - **Descrição**
```

>> - **Servidor de receção de correio POP3**:<br>- **Nome do host POP3**: para a **EUROPE**, introduza `pop.mail.ovh.net` ou `ssl0.ovh.net` . Para a **AMÉRICA/ÁSIA** introduza `pop.mail.ovh.ca`<br>- **Port**: 995<br>- **Tipo de segurança**: SSL/TLS<br>- **Nome de utilizador POP3** : o seu endereço de correio eletrónico completo<br>- **Palavra-passe POP3** : o do seu endereço de correio eletrónico
>> - **Servidor de receção de correio eletrónico SMTP**:<br>- **Nome do sistema anfitrião SMTP**: para a **EUROPE**, introduza `smtp.mail.ovh.net` ou `ssl0.ovh.net`. Para a **AMÉRICA/ÁSIA**, introduza `smtp.mail.ovh.ca`<br>- **Port**: 465<br>- **Tipo de segurança**: SSL/TLS<br>- **Nome de utilizador SMTP**: o seu endereço de correio eletrónico completo<br>- **Palavra-passe SMTP** : o do seu endereço de correio eletrónico
>>
>> Para finalizar a configuração, clique no botão " &#10003; "
>>
>> ![outlook android](images/outlook-app-android-add-step03-pop-eu.png){.thumbnail .w-400 .h-600}
>>

> [!warning]
>

> > Se, seguindo os passos de configuração acima, encontrar um problema de envio ou receção, vá para " [Editar definições existentes](#modify-settings )".

Reviewer: ```suggestion
> Se, seguindo os passos de configuração acima, encontrar um problema de envio ou receção, vá para " [Editar definições existentes](#modify-settings)".
```

>> - **Servidor Europe (de entrada)**: imap.mail.ovh.net **ou** ssl0.ovh.net
>> - **Servidor AMÉRICA/ÁSIA-PACÍFICO (de entrada)**: imap.mail.ovh.ca
>> - **Port** : 993
>> - **Tipo de segurança**: SSL/TLS
>>
> **Configuração POP**
>>
>> - **Nome de utilizador** : Introduza o endereço de e-mail **completo**
>> - **Palavra-passe** : Insira a palavra-passe do endereço de e-mail
>> - **Servidor Europe (de entrada)**: pop.mail.ovh.net **ou** ssl0.ovh.net
>> - **Servidor AMÉRICA/ÁSIA-PACÍFICO (de entrada)**: pop.mail.ovh.ca
>> - **Port** : 995
>> - **Tipo de segurança**: SSL/TLS

#### Definições de envio SMTP

Se necessita de inserir manualmente as definições **SMTP** nas preferências da conta para enviar uma mensagem de correio eletrónico, consulte as seguintes definições:

**Configuração SMTP**

- **Nome de utilizador** : Insira o endereço de e-mail **completo**
- **Palavra-passe**: Insira a palavra-passe do endereço de e-mail
- **Servidor Europe (de entrada)**: pop.mail.ovh.net **ou** ssl0.ovh.net
- **Servidor AMÉRICA/ÁSIA-PACÍFICO (de entrada)**: pop.mail.ovh.ca
- **Port** : 995
- **Tipo de segurança**: SSL/TLS

> [!primary]
>
> **Alterar a configuração**
>

> > Se o seu endereço de e-mail estiver configurado em **IMAP** e pretender alterar esta configuração para **POP**, deve eliminar a conta e recriá-la em **POP**. Consulte o capítulo " [Editar definições existentes](#modify-settings) " deste manual.

Reviewer: ```suggestion
> Se o seu endereço de e-mail estiver configurado em **IMAP** e pretender alterar esta configuração para **POP**, deve eliminar a conta e recriá-la em **POP**. Consulte o capítulo "[Editar definições existentes](#modify-settings)" deste manual.
```
